napari/_qt/widgets/qt_keyboard_settings.py

```diff
@@ -292,14 +292,52 @@ def _set_table(self, layer_str: str = ''):
             item.setFlags(Qt.ItemFlag.NoItemFlags)
             self._table.setItem(0, 0, item)
 
-    def _get_layer_actions(self):
+    def _get_potential_conflicting_actions(self):
+        """
+        Get all actions we want to avoid keybinding conflicts with.
+
+        If current selected keybinding group is a layer, return
+        the selected layer actions and viewer actions.
+        If the current selected keybinding group is viewer,
+        return actions from all layers.
```

Comment:
As I read the discussion in `_mark_conflicts` the docstring should mention that the order of actions is important, and the current group must be the first one.

Reply:
Also now that I read it again it should say:

```suggestion
        Get all actions we want to avoid keybinding conflicts with.

        If current selected keybinding group is a layer, return
        the selected layer actions and viewer actions.
        If the current selected keybinding group is viewer,
        return viewer actions and actions from all layers.
```